src/commands/general/msfsdiscord.ts

```diff
@@ -0,0 +1,14 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+
+const MSFS_DISCORD_LINK = 'https://discord.gg/BjEckUJUhe'
```

Comment:
Would prefer a permanent link to server

Reply:
https://discord.gg/msfs is the permalink to their discord server.